packages/ipfs-core/src/components/dag/export.js

```diff
@@ -0,0 +1,133 @@
+'use strict'
+
+const { CID } = require('multiformats/cid')
+const Block = require('multiformats/block')
+const { base58btc } = require('multiformats/bases/base58')
+const { CarWriter } = require('@ipld/car/writer')
+const withTimeoutOption = require('ipfs-core-utils/src/with-timeout-option')
+const log = require('debug')('ipfs:components:dag:import')
+const dagCbor = require('@ipld/dag-cbor')
+const raw = require('multiformats/codecs/raw')
+const json = require('multiformats/codecs/json')
+
+// blocks that we're OK with not inspecting for links
+/** @type {number[]} */
+const NO_LINKS_CODECS = [
+  dagCbor.code, // CBOR
```

Comment:
```suggestion
```

This should be `0x51` (cbor), not `0x71` (dag-cbor) like here, but it's probably best to just remove it since we don't have an official cbor encoder (yet, no big demand atm). We need to inspect links of dagCbor blocks, since it has links, but cbor blocks don't (or .. maybe they could do, :shrug:, it's not really clear what "cbor" means and its scope, but at least in Go we're going with super-minimal cbor with no tags or extras).

Reply:
Ah, that makes sense, I did wonder.